domain/actionSpace/urgentWishForChildren.unit.ts

```diff
@@ -88,7 +104,42 @@ describe("Urgent Wish for Children", () => {
         }
     });
 
-    it("should pay for dwelling", function () {});
+    describe("should buy dwelling", function () {
```

Comment:
[Detail] I know that this is more practical but it would be better to do a test:
- where the player has enough resources (but not exactly enough)
- where the player doesn't have enough resources but not 0

Reply:
https://github.com/BoredGamesWorkshop/caverna/pull/9/commits/3571395b5d11f8986f5924ad9f109ddba40339fc 😬